documentation/operators/src/faq.md

```diff
@@ -0,0 +1,7 @@
+<!---
```

Comment:
You can use a load of the stuff from here too https://nymtech.net/docs/nodes/troubleshooting.html

Reply:
This I am expecting to use as well. And separate it to troubleshooting and to FAQ, where also some basics on the active set etc shall be included.